docs/sphinx/whatsnew/v1.5.1.rst

```diff
@@ -0,0 +1,12 @@
+.. _whatsnew_151:
+
+v1.5.1 (DATE XX, 2021)
+======================
+
+Update pvlib dependency from ``pvlib>=0.6.0,<0.8.0`` to ``pvlib>=0.7.0,<0.9.0`` (:ghpull:`116`)
```

Comment:
The `ghpull` role is provided by the `sphinxcontrib_github_alt` extension, but previous whatsnews just use plaintext `#116` so I'll change this to plaintext for consistency if you want

Reply:
super cool! I didn't know about this 🚀